src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -223,7 +223,8 @@ def find_subscriptions_on_login(self,
             t_list = [s.tenant_id for s in subscriptions]
```

Comment:
Is it better to distinct t_list as there may be many dupplicated tenant_id?

Reply:
This it not necessary as `t_list` is for tenants which are not used to construct `bare_tenants`.